src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -75,7 +75,11 @@ public function getConfigTreeBuilder()
                     ->setDeprecated('The use of the `default_operation_path_resolver` has been deprecated in 2.1 and will be removed in 3.0. Use `path_segment_name_generator` instead.')
                     ->info('Specify the default operation path resolver to use for generating resources operations path.')
                 ->end()
-                ->scalarNode('name_converter')->defaultNull()->info('Specify a name converter to use.')->end()
+                ->scalarNode('name_converter')
+                    ->defaultNull()
+                    ->setDeprecated('Configuring "name_converter" within the "api_platform" node is deprecated and will be removed in 3.0. Please use the "serializer.name_converter" node instead.')
+                    ->info('Specify a name converter to use.')
```

Comment:
On a second though I don't think we should deprecate this because it introduces a hard link with symfony.

Reply:
I don't know if that makes sense. We're already in a Symfony bundle in this case. :laughing:

When using our bundle, we have a hard dependency on `symfony/framework-bundle`, no? Otherwise everything falls apart.